Crash when using Telegram notifications via shoutrrr

Docker on Ubuntu 20.04:
```
Server:
 Engine:
  Version:          19.03.11
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.13.12
  Git commit:       77e06fd
  Built:            Mon Jun  8 20:24:59 2020
  OS/Arch:          linux/amd64
  Experimental:     false
```
Using docker-compose with:
```
    environment:
     - WATCHTOWER_NOTIFICATIONS=shoutrrr
     - WATCHTOWER_NOTIFICATION_URL=telegram://<token>@telegram?channels=<channelid>
```
Causes this on startup:
```
time="2020-06-10T09:22:36Z" level=debug msg="Retrieving running containers"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x9fef3c]

goroutine 6 [running]:
github.com/containrrr/shoutrrr/pkg/services/telegram.sendMessageToAPI(0xc0002f1740, 0x1b, 0xc00008276c, 0xd, 0xc00034c390, 0x2e, 0xc00001e120, 0x0)
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200601144753-78bb9685bc2f/pkg/services/telegram/telegram.go:70 +0x21c
github.com/containrrr/shoutrrr/pkg/services/telegram.(*Service).sendMessageForChatIDs(0xc000308c60, 0xc0002f1740, 0x1b, 0xc0000386a0, 0xc000038780)
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200601144753-78bb9685bc2f/pkg/services/telegram/telegram.go:49 +0x9d
github.com/containrrr/shoutrrr/pkg/services/telegram.(*Service).Send(0xc000308c60, 0xc0002f1740, 0x1b, 0xc00000e018, 0x0, 0x0)
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200601144753-78bb9685bc2f/pkg/services/telegram/telegram.go:33 +0x8d
github.com/containrrr/shoutrrr/pkg/router.sendToService.func1(0xc00001e240, 0xca0640, 0xc000308c60, 0xc0002f1740, 0x1b, 0xc00000e018)
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200601144753-78bb9685bc2f/pkg/router/router.go:78 +0x4f
created by github.com/containrrr/shoutrrr/pkg/router.sendToService
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200601144753-78bb9685bc2f/pkg/router/router.go:78 +0xda
time="2020-06-10T09:22:37Z" level=info msg="Running a one time update."
time="2020-06-10T09:22:37Z" level=debug msg="Checking containers for updated images"
time="2020-06-10T09:22:37Z" level=debug msg="Retrieving running containers"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x9fef3c]

goroutine 8 [running]:
github.com/containrrr/shoutrrr/pkg/services/telegram.sendMessageToAPI(0xc0000917a0, 0x1b, 0xc0000de52c, 0xd, 0xc0000ea780, 0x2e, 0xc000218f00, 0x0)
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200601144753-78bb9685bc2f/pkg/services/telegram/telegram.go:70 +0x21c
github.com/containrrr/shoutrrr/pkg/services/telegram.(*Service).sendMessageForChatIDs(0xc0000859c0, 0xc0000917a0, 0x1b, 0xc00003c6a0, 0xc00003c780)
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200601144753-78bb9685bc2f/pkg/services/telegram/telegram.go:49 +0x9d
github.com/containrrr/shoutrrr/pkg/services/telegram.(*Service).Send(0xc0000859c0, 0xc0000917a0, 0x1b, 0xc00000e418, 0x0, 0x0)
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200601144753-78bb9685bc2f/pkg/services/telegram/telegram.go:33 +0x8d
github.com/containrrr/shoutrrr/pkg/router.sendToService.func1(0xc000219020, 0xca0640, 0xc0000859c0, 0xc0000917a0, 0x1b, 0xc00000e418)
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200601144753-78bb9685bc2f/pkg/router/router.go:78 +0x4f
created by github.com/containrrr/shoutrrr/pkg/router.sendToService
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200601144753-78bb9685bc2f/pkg/router/router.go:78 +0xda
time="2020-06-10T09:22:38Z" level=info msg="Running a one time update."
time="2020-06-10T09:22:38Z" level=debug msg="Checking containers for updated images"
time="2020-06-10T09:22:38Z" level=debug msg="Retrieving running containers"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x9fef3c]

goroutine 26 [running]:
github.com/containrrr/shoutrrr/pkg/services/telegram.sendMessageToAPI(0xc0002f1740, 0x1b, 0xc00008276c, 0xd, 0xc00034c390, 0x2e, 0xc000219d40, 0x0)
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200601144753-78bb9685bc2f/pkg/services/telegram/telegram.go:70 +0x21c
github.com/containrrr/shoutrrr/pkg/services/telegram.(*Service).sendMessageForChatIDs(0xc000308ce0, 0xc0002f1740, 0x1b, 0xc0000386a0, 0xc000038780)
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200601144753-78bb9685bc2f/pkg/services/telegram/telegram.go:49 +0x9d
github.com/containrrr/shoutrrr/pkg/services/telegram.(*Service).Send(0xc000308ce0, 0xc0002f1740, 0x1b, 0xc00009a568, 0x0, 0x0)
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200601144753-78bb9685bc2f/pkg/services/telegram/telegram.go:33 +0x8d
github.com/containrrr/shoutrrr/pkg/router.sendToService.func1(0xc000219e60, 0xca0640, 0xc000308ce0, 0xc0002f1740, 0x1b, 0xc00009a568)
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200601144753-78bb9685bc2f/pkg/router/router.go:78 +0x4f
created by github.com/containrrr/shoutrrr/pkg/router.sendToService
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200601144753-78bb9685bc2f/pkg/router/router.go:78 +0xda
time="2020-06-10T09:22:39Z" level=info msg="Running a one time update."
time="2020-06-10T09:22:39Z" level=debug msg="Checking containers for updated images"
time="2020-06-10T09:22:39Z" level=debug msg="Retrieving running containers"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x9fef3c]

goroutine 6 [running]:
github.com/containrrr/shoutrrr/pkg/services/telegram.sendMessageToAPI(0xc0002f18a0, 0x1b, 0xc00008276c, 0xd, 0xc00034c3f0, 0x2e, 0xc00001e120, 0x0)
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200601144753-78bb9685bc2f/pkg/services/telegram/telegram.go:70 +0x21c
github.com/containrrr/shoutrrr/pkg/services/telegram.(*Service).sendMessageForChatIDs(0xc00030acc0, 0xc0002f18a0, 0x1b, 0xc0000386a0, 0xc000038780)
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200601144753-78bb9685bc2f/pkg/services/telegram/telegram.go:49 +0x9d
github.com/containrrr/shoutrrr/pkg/services/telegram.(*Service).Send(0xc00030acc0, 0xc0002f18a0, 0x1b, 0xc00000e018, 0x0, 0x0)
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200601144753-78bb9685bc2f/pkg/services/telegram/telegram.go:33 +0x8d
github.com/containrrr/shoutrrr/pkg/router.sendToService.func1(0xc00001e240, 0xca0640, 0xc00030acc0, 0xc0002f18a0, 0x1b, 0xc00000e018)
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200601144753-78bb9685bc2f/pkg/router/router.go:78 +0x4f
created by github.com/containrrr/shoutrrr/pkg/router.sendToService
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200601144753-78bb9685bc2f/pkg/router/router.go:78 +0xda
time="2020-06-10T09:22:40Z" level=info msg="Running a one time update."
time="2020-06-10T09:22:40Z" level=debug msg="Checking containers for updated images"
time="2020-06-10T09:22:40Z" level=debug msg="Retrieving running containers"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x9fef3c]

goroutine 21 [running]:
github.com/containrrr/shoutrrr/pkg/services/telegram.sendMessageToAPI(0xc000025500, 0x1b, 0xc00032252c, 0xd, 0xc0000267b0, 0x2e, 0xc000218f00, 0x0)
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200601144753-78bb9685bc2f/pkg/services/telegram/telegram.go:70 +0x21c
github.com/containrrr/shoutrrr/pkg/services/telegram.(*Service).sendMessageForChatIDs(0xc00000cf80, 0xc000025500, 0x1b, 0xc0000386a0, 0xc000038780)
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200601144753-78bb9685bc2f/pkg/services/telegram/telegram.go:49 +0x9d
github.com/containrrr/shoutrrr/pkg/services/telegram.(*Service).Send(0xc00000cf80, 0xc000025500, 0x1b, 0xc00009a410, 0x0, 0x0)
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200601144753-78bb9685bc2f/pkg/services/telegram/telegram.go:33 +0x8d
github.com/containrrr/shoutrrr/pkg/router.sendToService.func1(0xc000219020, 0xca0640, 0xc00000cf80, 0xc000025500, 0x1b, 0xc00009a410)
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200601144753-78bb9685bc2f/pkg/router/router.go:78 +0x4f
created by github.com/containrrr/shoutrrr/pkg/router.sendToService
	/go/pkg/mod/github.com/containrrr/shoutrrr@v0.0.0-20200601144753-78bb9685bc2f/pkg/router/router.go:78 +0xda
```

--- a/pkg/services/telegram/telegram.go
+++ b/pkg/services/telegram/telegram.go
@@ -25,7 +25,7 @@ type Service struct {
 }
 
 // Send notification to Telegram
-func (service *Service) Send(message string, params *types.Params) error {
+func (service *Service) Send(message string, _ *types.Params) error {
 	if len(message) > maxlength {
 		return errors.New("message exceeds the max length")
 	}
@@ -60,14 +60,17 @@ func (service *Service) GetConfig() *Config {
 
 func sendMessageToAPI(message string, channel string, apiToken string) error {
 	postURL := fmt.Sprintf("%s%s/sendMessage", apiBase, apiToken)
-	jsonData, _ := json.Marshal(
+	jsonData, err := json.Marshal(
 		JSON{
 			Text: message,
 			ID:   channel,
 		})
+	if err != nil {
+		return err
+	}
 
 	res, err := http.Post(postURL, "application/jsonData", bytes.NewBuffer(jsonData))
-	if res.StatusCode != http.StatusOK {
+	if err == nil && res.StatusCode != http.StatusOK {
 		return fmt.Errorf("failed to send notification to \"%s\", response status code %s", channel, res.Status)
 	}
 	return err

